packages/blocks/src/note-block/note-model.ts

```diff
@@ -65,6 +65,9 @@ type NoteEdgelessProps = {
   };
   collapse?: boolean;
   collapsedHeight?: number;
+  width?: number;
```

Comment:
We already have `xywh`, how to clarify the relationship between these fields? I hope there should not be duplicates.

Reply:
maybe I can just calculate this using xywh and scale

Reply:
@doodlewind 
done